cmd/root.go

```diff
@@ -25,6 +26,20 @@ func getEnvDefault(env string, def string) string {
 	return def
 }
 
+func lengthofLongest(files []os.FileInfo) int {
+	m := 0
+	for _, e := range files {
+		if len(e.Name()) > m {
+			m = len(e.Name())
+		}
+	}
+	return m
+}
+
+func leftPad(str string, length int) string {
+	return strings.Repeat(" ", length-len(str)+1) + str
```

Comment:
This is likely better handled with a `fmt.Printf`.  Likely I would merge this into the above function and have it return a printf string to handle the formatting.